dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/PullRequests.java

```diff
@@ -41,5 +49,33 @@ public void prepareToCreate(final Namespace ns,
         }
         ResultsProcessor.processResults(skippedRepos, exceptions, log);
     }
+
+    /**
+     * Check if the repository is onboarded to Renovate. The signal we are looking for are
+     * (1) The presence of a file where renovate configurations are stored in the repository
+     * (2) Ensuring that the file does not have the key "enabled" set to "false"
+     * @param filePaths the list that contains all the names of the files that we are searching for in the repo
+     * @param fork A GitHubContentToProcess object that contains the fork repository that is under process
+     * @return true if the file is found in the path specified and is not disabled, false otherwise
+     */
+    protected boolean isRenovateEnabled(List<String> filePaths, GitHubContentToProcess fork) throws IOException {
+        for (String filePath : filePaths) {
+            try {
+                GHContent fileContent = fork.getParent().getFileContent(filePath);
+                JSONObject json;
+                try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(fileContent.read()))) {
+                    JSONTokener tokener = new JSONTokener(bufferedReader);
+                    json = new JSONObject(tokener);
+                    //If the file has the key 'enabled' set to false, it indicates that while the repo has been onboarded to renovate, it has been disabled for some reason
+                    return json.optBoolean("enabled", true);
```

Comment:
So the users need to delete this key altogether to get DFIU PRs back? true/false both mean we skip DFIU PRs?

Reply:
Here are the scenarios
- Renovate config file not found - Send DFIU PR
- Renovate config file found and file does not have key 'enabled' - Skip DFIU PR
- Renovate config file found and file has the key 'enabled' - return the boolean value for that key. If set to "true", skip DFIU PR, if set to "false", send DFIU PR

This key is usually used when a repo onboards to Renovate but then decides to off board by adding the key `enabled`: false.

Reply:
"Renovate config file found and file does not have key enabled - Skip DFIU PR" Should we do this? the config file doesn't have this key, doesn't that mean it's enabled? the default is set to true https://docs.renovatebot.com/configuration-options/#enabled

Reply:
and i don't think the method works like that at the moment. https://github.com/stleary/JSON-java/blob/master/src/main/java/org/json/JSONObject.java#L1143
The method will return true if the file doesn't have the key enabled and we'll skip the PR

Reply:
or am I overlooking something?

Reply:
> "Renovate config file found and file does not have key enabled - Skip DFIU PR" Should we do this? the config file doesn't have this key, doesn't that mean it's enabled? the default is set to true https://docs.renovatebot.com/configuration-options/#enabled

Yes, it does mean it is enabled. So we will skip DFIU because we expect to receive PRs from Renovate

Reply:
I see. This comment confused me a little, I thought we wanted to skip sending the PR in this case. 
//If the file has the key 'enabled' set to false, it indicates that while the repo has been onboarded to renovate, it has been disabled for some reason.